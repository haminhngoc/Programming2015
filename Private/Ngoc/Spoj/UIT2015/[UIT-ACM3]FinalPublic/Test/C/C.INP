42 18468
6335 26501
19170 15725
1 1000000
26963 24465
5706 28146
23282 16828
9962 492
1000000 2
1000000 1000000

