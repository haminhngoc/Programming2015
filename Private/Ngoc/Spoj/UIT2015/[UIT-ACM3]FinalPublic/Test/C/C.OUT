16
1285
205
1
23130
4527
8972
8608
951425
485401

